containers/tefca-viewer/src/styles/custom-styles.scss

```
@@ -270,6 +262,17 @@ body {
   justify-content: center;
   margin: 0 auto;
   max-width: 1000px;
+
+  .usa-table--borderless {
+    tr {
+      &:hover td {
+        background-color: #D9E8F680;
```

```suggestion
        background-color: #d9e8f6;
```

I think this is the color from the Figma file

---

It totally is, I have literally no idea where the random 80 came from